prow/plugins/trigger/ic_test.go

```diff
@@ -48,7 +49,6 @@ func TestHandleIssueComment(t *testing.T) {
 		Branch        string
 		ShouldBuild   bool
 		ShouldReport  bool
-		HasOkToTest   bool
 		IsOkToTest    bool
```

Comment:
Given the way this function works now I think removing `IsOkToTest` and replacing it with:

```golang
StartLabels []github.Label
EndLabels []github.Label
```

Will be an easier to read test case.